spec/models/post_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(title: 'My Post', text: 'This is my post content') }
+
```

Comment:
- [ ] I suggest that any custom methods in the models should have a corresponding unit test so please ensure that you add a test for the `update_user_posts_counter` and `recent_comments` in your post model :pray: